### ISSUE TITLE
Populate FEC Compliance Map for 2018

### ISSUE THREAD

**Issue by dorothyyeager** [18F/openFEC-web-app#2337]
The Information Division maintains the compliance map at https://transition.fec.gov/info/ElectionDate/ which is used by the regulated community to get information on filing FEC reports, including pre-election reports that depend on each state's primary election date. Normally the map is populated for an election year during the fall of the non-election year. The process is started with Information Division contacting ITD to do a "data dump" from "Track Committees" to load and create the map for the new year.

Although the map will remain on the transition site during this election year, we need to make sure we factor the task of dumping the data into our fall website planning, as the work needs to take place this quarter so that the Information division will be able to complete its work on the map by January when new filing dates for the year are announced. Kathy Carothers in Information is the lead person for the compliance map and can answer any questions about what's needed in terms of data. She's sent an email already to the people who worked on this in the past.

Next to last step: set up HTML page for the map.
Last step: remove from script publishing of 2018 election year for compliance map.

**Comment by AmyKort** [comment]
How soon do we need to get started on this?

**Comment by dorothyyeager** [comment]
I'm told that usually the process begins around now (October non-election year) but should be underway no later than the beginning of November.

**Comment by lizkurland** [comment]
The goal is to have the map and all the state pages populated and updated so they are ready to go live on 1/2/18.

**Comment by patphongs** [comment]
@lizkurland @dorothyyeager @AmyKort I've set up the new 2018 compliance map pages on the Contribute staging server. It's all ready for info to start populating. I have also stopped the script from automatically publishing any changes in the `/info/ElectionDate` directory. So if there's any special elections for 2017, just email me to manually push it to production for you. Once the 2018 pages are done, I can reactivate the automated script for that folder.
